src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java

```diff
@@ -674,8 +674,10 @@ private CommandLine buildOptions(String[] args) {
         OPTION_DEBUG_MODE,
         "debug-mode",
```

Comment:
Should we consider changing the name of the flag?

Reply:
Yes, good point. I've moved that out of this change and will make a separate PR for that.